qa/tasks/cephfs/test_volumes.py

```diff
@@ -5845,7 +5845,7 @@ def test_subvolume_snapshot_info_if_clone_pending_for_no_group(self):
         self._fs_cmd("subvolume", "snapshot", "create", self.volname, subvolume, snapshot)
 
         # insert delay at the beginning of snapshot clone
-        self.config_set('mgr', 'mgr/volumes/snapshot_clone_delay', 5)
+        self.config_set('mgr', 'mgr/volumes/snapshot_clone_delay', 15)
```

Comment:
Instead of moving this commit on a separate PR, this should be kept as a separate commit on this PR because this adjustment is needed only  due to new code added on this PR. This will not only preserve the context of this commit but also keep make it's QA run and backporting easy

Also, this commit should not be squashed into the commit that adds tests for clone progress reporter, a separate commit message allows to mention exactly  why this change is needed.

Ofcourse, if we keep it here, commit message needs to be changed,